index.html

```diff
@@ -252,8 +252,11 @@ <h2>Contact Me</h2>
       needs coding. I’d love to help with it </p>
     </div>
     <form action="https://formspree.io/f/xgeqpkgk" method="post">
+      <label for="text" hidden="hidden">name:</label>
       <input type="text" name="text" id="text" placeholder="Enter name" maxlength="30" required>
+      <label for="email" hidden="hidden">email:</label>
       <input type="email" name="email" id="email" placeholder="Enter email" required>
+      <label for="formMessage" hidden="hidden">message:</label>
       <textarea name="formMessage" id="message" placeholder="Write your message here" maxlength="500"
```

Comment:
- [ ] Nice job adding labels! 💯 However, the approach you chose to hide them not only works visually but for assistive technologies such as screen readers. Please, check [this article](https://www.w3.org/WAI/tutorials/forms/labels/#hiding-label-text) on how to hide labels properly, and use what is most suited to this use case! Consider changing this! 🤓 🛠️